.env.example

```diff
@@ -19,5 +19,11 @@ VERIFIED_EMAIL=
 CONTACT_EMAIL_TEMPLATE=
 HOST_HTTP_OR_HTTPS=http
 IMAGE_PATH=../public/images/
+
+ENVIRONMENT=localhost
+PORT=3000
```

Comment:
There is already a variable called `PORT` (line 16). Please, remove older one and keep this one with its default value